.github/branchcleanup.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Branch Cleanup
+# This workflow is triggered on all closed pull requests.
+# However the script does not do anything it a merge was not performed.
```

Comment:
s/it/if/
Is the workflow the same as the script?

Maybe simpler wording.

This workflow is trigged on all closed pull requests, does something only if a merge is performed.

Reply:
Fixed the wording.